test/lib/jdk/test/lib/security/DerUtils.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Is this test change supposed to be a part of this fix?

Reply:
Yes, the change simplifies `checkAlg` calls so they don't need to convert `KnownOIDs` or `String` to `ObjectIdentifier` first.